src/awkward/operations/ak_from_raggedtensor.py

```diff
@@ -55,6 +78,18 @@ def _impl(array):
     return ak.Array(_recursive_call(content, offsets_arr, 0))
 
 
+def _tensor_to_np_or_cp(array, device):
+    import tensorflow as tf
+
+    if "GPU" in device:
```

Comment:
Is it always capitalized like that? (I.e. what are all the _possible_ values of `device`?) Does it always start with `"GPU"`? If so, you want to narrow down the match condition as much as possible:

```suggestion
    if device.startswith("GPU"):
```

(Does it always start with `"GPU-"`?)

Reply:
this will work: `device.endswith("GPU", 0, -2)` thanks!